make.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+run () {
+    echo "Running application..."
+    go run -tags sdl . $1
```

Comment:
Fix handling of filenames with spaces:
```suggestion
    go run -tags sdl . "$1"
```